src/core/server/saved_objects/migrations/core/build_active_mappings.ts

```diff
@@ -34,6 +34,7 @@
 
 import crypto from 'crypto';
 import { cloneDeep, mapValues } from 'lodash';
+import { Config } from 'packages/osd-config/target';
```

Comment:
shall we merge https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6012 to this one or will have another cherry-pick for 6012?

Reply:
Will have another cherry-pick.